buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-Any-Platform

```diff
@@ -36,10 +37,14 @@ timestamps {
 
             buildFile = load 'buildenv/jenkins/common/pipeline-functions'
             SHAS = buildFile.get_shas(OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO, OPENJ9_BRANCH, OMR_REPO, OMR_BRANCH, VENDOR_TEST_REPOS_MAP, VENDOR_TEST_BRANCHES_MAP, VENDOR_TEST_SHAS_MAP)
+            BUILD_NAME="Build-JDK${SDK_VERSION}-${SPEC}"
+            if (params.AUTOMATIC_GENERATION){
```

Comment:
I _think_ we want to always generate unless explicitly set to false. So maybe
```
if (params.AUTOMATIC_GENERATION != false) {
```
?